sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -17,10 +17,14 @@
 
 """Tests common to all coder implementations."""
 from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
<!--new_thread; commit:57a73055b9447e7a1fe6ad674673254d9a614eeb; resolved:0-->
Which of these is really necessary?  Do we use print in this file?

Reply:
All three have been added to the top of each updated module. This is safest to avoid regression before we add full python 3 support.